### PR TITLE
Feat 4 buttons

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the buttons component:
- Button Variants - default, rounded and curved corners
- Primary Button

image

- Secondary Button
- Link Button

image

- Icon Button
- Floating Action Button

image

### CODE REVIEW

**Review by anshulraheja — APPROVED** [review]
LGTM

File: stylesheets/colors.css [hunk 1]
```diff
@@ -1,41 +1,62 @@
...
-    --border-red: #b10e0e;
-    --border-green: #5da516;
-    --border-grey: #797878;
+    --secondary-hover: #bd9390;
+    --secondary-disabled: #d3bebc;
+    --secondary-disabled-text: #9b9494;
 
+    --default-red: #af2b21;
+    --default-green: #6ab91b;
+    --default-grey: #969494;
```
Comment (pankaj-prog) [inline stylesheets/colors.css:24]: Use uppercase letters for hard coded values

**Review by pankaj-prog — COMMENTED** [review]
Code is clean and naming is also declarative.

- I never thought of styling fab class directly. Got to learn this.

**Review by j24m — APPROVED** [review]
Everything looks perfect to me, the code and the UI. GTM!! :)

**Comment by SinchanaMS** [comment]
Thank you for the reviews. Merging to dev.

**Pull request merged**